### PR TITLE
complain -> warn mode

### PR DESCRIPTION
**Pull request by haripriya9647**
#43718

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Thanks.

Why should we make this change?

File: content/en/docs/tutorials/security/apparmor.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ permissions to confine programs to a limited set of resources. AppArmor can be c
 application to reduce its potential attack surface and provide greater in-depth defense. It is
 configured through profiles tuned to allow the access needed by a specific program or container,
 such as Linux capabilities, network access, file permissions, etc. Each profile can be run in either
-*enforcing* mode, which blocks access to disallowed resources, or *complain* mode, which only reports
+*enforcing* mode, which blocks access to disallowed resources, or *warn* mode, which only reports
```
Comment (Ritikaa96) [inline content/en/docs/tutorials/security/apparmor.md:19]:
```suggestion
*enforced* mode, which blocks access to disallowed resources, or *complaining* mode, which only reports
```
As per https://ubuntu.com/server/docs/security-apparmor the only two modes are: Enforced/confined and Complaining/learning. There is no warn mode. @haripriya9647  can you elaborate on your change?

**Comment by dipesh-rawat** [comment]
Hey @haripriya9647, whenever you get a chance, could you please take a look at the reviewers' feedback and respond to them? Thanks!

**Comment by sftim** [comment]
No response to https://github.com/kubernetes/website/pull/44851#pullrequestreview-1837605833

/close

@haripriya9647, I don't think this change is useful. However, we'd be happy to help you find other ways to contribute to Kubernetes.

**Pull request closed without merging**